test/Passes/mlp-all.mlir

```diff
@@ -1,107 +0,0 @@
-// RUN: tpp-opt %s -bufferize -convert-linalg-to-tpp | FileCheck %s
-
-#map0 = affine_map<(d0, d1) -> (d1)>
```

Comment:
Is there a multi-layer MLP test on tensors?

Reply:
Hmm looks like there isn't. This one was removed because it was a poorly written kernel and not relevant anymore.
I'll add a separate 3 layer mlp test.

Reply:
(They are a few in benchmarks but there should be one in `test` for clarity)

Reply:
Added new mlp3-3layers test on tensors